winml/lib/Api.Ort/OrtEngine.cpp

```diff
@@ -0,0 +1,272 @@
+#include "pch.h"
+
+#include "OrtEngine.h"
```

Comment:
every other file here has "Onnxruntime" prefix - do you want to be consistent here? "OnnxruntimeEngine"

Reply:
this whole file should not exist... was renamed... but i forgot to delete it... gone now!

---
In reply to: [371458507](https://github.com/microsoft/onnxruntime/pull/2891#discussion_r371458507) [](ancestors = 371458507)